src/ows/src/main/java/org/geoserver/ows/util/OwsUtils.java

```diff
@@ -206,19 +206,20 @@ public static Method getter(Class clazz, String property, Class type) {
 
     /**
      * Reflectivley retreives a propety from a java bean.
+     * @param <T>
      *
      * @param object The java bean.
      * @param property The property to retreive.
      * @param type Teh type of the property to retreive.
      *
      * @return The property, or null if it could not be found..
      */
-    public static Object property(Object object, String property, Class type) {
+    public static <T> T property(Object object, String property, Class<T> type) {
```

Comment:
Should be safe.  Technically I suppose someone somewhere might be storing a class in a raw generic variable and then passing it in, but it seems extremely unlikely.  I can skip this change and cast the expressions using it instead when backporting if anyone thinks this would be a problem.